doc/examples/data/plot_specific.py

```diff
@@ -55,9 +55,52 @@
 
 fig, axes = plt.subplots(4, 5, figsize=(20, 20))
 ax = axes.ravel()
-images = data.lfw_subset()
+lfs_images = data.lfw_subset()
 for i in range(20):
-    ax[i].imshow(images[90+i], cmap=plt.cm.gray)
-    ax[i].axis('off')
+    ax[i].imshow(lfs_images[90 + i], cmap=plt.cm.gray)
+    ax[i].axis("off")
```

Comment:
I think renaming these is more expressive and an improvement. However why `lfs_` and not `lfw_` like the corresponding function?

Reply:
Thats a typo I made, I guess the "w" and the "s" are very close together on my keyboard and then I copy pasted this error. I'll fix ir right now.